### ISSUE TITLE
Version 3.0.0 removes newline at EOF

### ISSUE THREAD

**Issue by luminoso** [Gudahtt/prettier-plugin-sort-json#168]
When running prettier-plugin-sort-json 3.0.0 with prettier 3.0.2, it removes newline at EOF.

Example:
`test.json`: (notice the newline at EOF)
```
{
  "name": "foobar"
}

```

after running `npx prettier -w test.json` it edits the file to:

```
{
  "name": "foobar"
}
```

Disabling the plugin prettier and re-running prettier, it restores the newline at EOFs.

Is this expected?

`.prettierrc`:
```
{
  "jsonRecursiveSort": true,
  "plugins": ["prettier-plugin-sort-json"]
}

```

`package.json`:
```
{
  "devDependencies": {
    "prettier": "^3.0.2",
    "prettier-plugin-sort-json": "^3.0.0"
  }
}
```

**Comment by Gudahtt** [comment]
Thanks for the report! No this isn't expected.

**Comment by Gudahtt** [comment]
This should be fixed in v3.0.1. Let me know if that works!
